07-18 18:19:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAY_EVOLUTIONS" not found; SQL statement:
select id, hash, apply_script, revert_script, state, last_problem from play_evolutions where state like 'applying_%' [42102-175]
07-21 17:34:38 jdbc[26]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:142)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:115)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:115)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:113)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:113)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 43 more
07-21 17:43:09 jdbc[47]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "USER_ID"; expected "UPDATE, {"; SQL statement:
USER_ID [42001-175]
07-21 21:01:18 jdbc[177]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:142)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:115)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:115)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:113)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:113)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 43 more
07-21 21:01:18 jdbc[177]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:656)
	at org.h2.server.web.WebApp.login(WebApp.java:905)
	at org.h2.server.web.WebApp.process(WebApp.java:209)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:168)
	at org.h2.server.web.WebThread.process(WebThread.java:138)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:142)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:115)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:115)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:113)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:113)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 43 more
07-21 21:01:18 jdbc[177]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:99)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:117)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:656)
	at org.h2.server.web.WebApp.login(WebApp.java:905)
	at org.h2.server.web.WebApp.process(WebApp.java:209)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:168)
	at org.h2.server.web.WebThread.process(WebThread.java:138)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:142)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:115)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:115)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:113)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:113)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 43 more
07-23 11:36:47 jdbc[5]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.jolbox.bonecp.BoneCP.obtainRawInternalConnection(BoneCP.java:351)
	at com.jolbox.bonecp.BoneCP.<init>(BoneCP.java:416)
	at com.jolbox.bonecp.BoneCPDataSource.getConnection(BoneCPDataSource.java:120)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:240)
	at play.api.db.BoneCPPlugin$$anonfun$onStart$1.apply(DB.scala:238)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at play.api.db.BoneCPPlugin.onStart(DB.scala:238)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.Play$.start(Play.scala:90)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:142)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:115)
	at scala.Option.map(Option.scala:145)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:115)
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:113)
	at scala.util.Success.flatMap(Try.scala:200)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:113)
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:105)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	... 43 more
